Article/Article.js

```diff
@@ -1,26 +1,30 @@
 // Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
-    // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
-    // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
-  }
+    constructor(domElement) {
+        // assign this.domElement to the passed in domElement
+        this.domElement = domElement;
+        // create a reference to the ".expandButton" class. 
+        this.expandButton = this.domElement.querySelector('.expandButton')
+            // Using your expandButton reference, update the text on your expandButton to say "expand"
+        this.expandButton.textContent = 'expand';
+        //.textContent = "expand";
 
-  expandArticle() {
-    // Using our reference to the domElement, toggle a class to expand or hide the article.
+        // Set a click handler on the expandButton reference, calling the expandArticle method.
+        this.expandButton.addEventListener('click', () => this.expandArticle(event));
 
-  }
+    }
+
+    expandArticle(event) {
+        // Using our reference to the domElement, toggle a class to expand or hide the article.
+        console.log();
+        this.domElement.classList.toggle('article-open');
+
+    }
 }
 
 // START HERE: Select all classes named ".article" and assign that value to the articles variable
-let articles;
+let articles = document.querySelectorAll(".article");
 
 // Use .map() to iterate over the articles array and create a new instance of Article by passing in each article as a parameter to the constructor.
-articles = Array.from(articles);
-
+articles = Array.from(articles).map((domElement) => { return new Article(domElement); });
```

Comment:
- Could also be written as `articles = Array.from(articles).map(domElement => new Article(domElement));` since it's a one-liner